sosia/classes/original.py

```diff
@@ -1,5 +1,7 @@
 """Main module of `sosia` containing the `Original` class."""
 
+from __future__ import annotations
```

Comment:
Why do we need this?

Reply:
To be able to use `Original` as type

Reply:
We can use `from typing_extensions import Self`.  [From 3.11 onwards we can use](https://peps.python.org/pep-0673/) `from typing import Self`.